### PR TITLE
log_softmax #15916

### PR DESCRIPTION
**Pull request by maher-mohsen**
- [ ] #15916

### CODE REVIEW

**Comment by maher-mohsen** [comment]
@AnnaTz could you review please, and help me to resolve conflicts cause i have no idea how can i reslove it

**Comment by AnnaTz** [comment]
> * [ ]  [log_softmax #15916](https://github.com/unifyai/ivy/issues/15916)

Hi @maher-mohsen, please link this PR to issues in the corresponding paddle ToDo list. 
You are supposed to leave this kind of comments under the ToDo, not here. And since you are adding more than one functions you will need to add more than one comments.

**Comment by AnnaTz** [comment]
> > * [ ]   [log_softmax #15916](https://github.com/unifyai/ivy/issues/15916)
> 
> Hi @maher-mohsen, please link this PR to issues in all corresponding paddle ToDo lists. Since you are adding more than one functions you will need to open more issues.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py [hunk 1]
```diff
@@ -248,4 +277,5 @@ def test_paddle_divide(
         on_device=on_device,
         x=x[0],
         y=x[1],
+        y=x[1],
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:280]: I missed this line when merging conflicts. Please remove the duplicate.

**Comment by AnnaTz** [comment]
> > * [ ]    [log_softmax #15916](https://github.com/unifyai/ivy/issues/15916)
> 
> Hi @maher-mohsen, please link this PR to issues in all corresponding paddle ToDo lists. Since you are adding more than one functions you will need to open more issues.

Nevermind ignore this. I realized you didn't add `log` and `divide` yourself, it just seemed like you did due to your incorrect merge.

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py [hunk 2]
```diff
@@ -192,6 +192,35 @@ def test_paddle_asin(
...
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        x=x[0],
+    )
+
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py:222]: The test is passing! 
Only thing to fix is a lint formatting error. Please have a look at `lint / Check formatting / check-formatting (pull_request)` in the CI checks at the bottom of the PR.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by maher-mohsen** [comment]
@AnnaTz could you help me to fix current formatting issue because I don't understand it .

**Comment by maher-mohsen** [comment]
@AnnaTz all done, ...ready to merge now

**Pull request merged**